prism/Makefile

```diff
@@ -264,27 +284,18 @@ ifeq ($(OSTYPE),darwin)
 	CLASSPATHSEP = :
 endif
 
-###############
-# Directories #
-###############
+export CFLAGS CXXFLAGS LDFLAGS JFLAGS LIBPREFIX LIBSUFFIX
```

Comment:
Is there some logic here as to which of the vars defined above this are exported, and which are passed to sub-Makefiles in other ways? There is a problem, currently, that `EXE` is no longer passed down within `make_dirs` (this becomes evident only on Cygwin).

Reply:
`EXE` shouldn't be exported (at the moment I'm just exporting standard make variables like compiler paths and flags plus widely-used PRISM-specific variables like `OSTYPE`) - `EXE` was accidentally removed from the `make` invocation in `make_dirs` in 0b21eae9, now added back in c134e7fd.